otterdog/eclipse-keyple.jsonnet

```diff
@@ -548,7 +548,7 @@ orgs.newOrg('iot.keyple', 'eclipse-keyple') {
         },
       ],
     },
-    orgs.newRepo('keyple-plugin-storage-card-java-api') {
+    orgs.newRepo('keyple-plugin-storagecard-java-api') {
```

Comment:
renaming requires the use of aliases. See https://otterdog.readthedocs.io/en/latest/userguide/renaming/ for details

Reply:
ah yes, sorry for the oversight

Reply:
now, both the name and the alias are the same :)

Reply:
my bad, misread